instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Comment from './Comment';
+import CommentInput from './CommentInput';
+
+
+class CommentSection extends React.Component{
+    constructor(props){
+        super(props);
+        this.state = {
+            comments: props.comments,
+            comment:'',
+            date: props.date
```

Comment:
Line 11 - I'd advise renaming this variable to something that more explicitly indicates that it is an input string (perhaps something like `commentInput` or `newCommentText`).